### ISSUE TITLE
dotnet outputs executable files on linux

### ISSUE THREAD

**Issue by colemickens** [dotnet/cli#4881]
## Steps to reproduce
`dotnet new`

## Expected  behavior
Read/write files are created

## Actual behavior
Read/write/executable files are created:

```
total 16
drwxr-xr-x  2 cole cole 4096 Dec  1 05:44 .
drwxr-xr-x 22 cole cole 4096 Dec  1 05:44 ..
-rwxr--r--  1 cole cole  124 Nov  2 03:51 Program.cs
-rwxr--r--  1 cole cole  808 Nov  4 20:31 repro.csproj
```

## Environment data
`dotnet --info` output:
.NET Command Line Tools (1.0.0-preview3-004056)

Product Information:
 Version:            1.0.0-preview3-004056
 Commit SHA-1 hash:  ccc4968bc3

Runtime Environment:
 OS Name:     debian
 OS Version:  8
 OS Platform: Linux
 RID:         debian.8-x64
```

**Comment by svick** [comment]
This looks like a duplicate of https://github.com/dotnet/cli/issues/4260, whose root cause (https://github.com/dotnet/corefx/issues/13342) has already been fixed. So I think this should be fixed in the next version.

**Comment by piotrpMSFT** [comment]
CLI is currently built atop the latest LTS corefx [e.g. 1.0.*]. The fix to #4260 went into the `Current` train of CoreFx, 1.1.*, which the CLI is not planned to target. We expect to get that fix once LTS moves to corefx 2.0.* and the fix is available on that train.

We are looking at some approaches for getting fixes from CoreFx to CLI faster but are not yet ready to change the plan.

**Comment by livarcocc** [comment]
I am going to close this one in favor of #4260. I don't see the point of using two issues to track the same problem.
